prov/shm/src/smr_msg.c

```diff
@@ -51,120 +51,92 @@ static inline uint16_t smr_convert_rx_flags(uint64_t fi_flags)
 }
 
 static inline struct smr_ep_entry *smr_get_recv_entry(struct smr_ep *ep,
-		fi_addr_t addr, uint64_t flags)
+		const struct iovec *iov, size_t count, fi_addr_t addr,
```

Comment:
I would drop the inline from this.  The call is a little long, and the compiler may ignore it anyway.